indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -294,7 +291,7 @@ public void setConf(Configuration config)
 
   public static class IndexGeneratorReducer extends Reducer<BytesWritable, Writable, BytesWritable, Text>
   {
-    private HadoopDruidIndexerConfig config;
+    protected HadoopDruidIndexerConfig config;
```

Comment:
this is only required for overrides in `LegacyIndexGeneratorReducer`, so I want to limit its use as much as possible.